### PR TITLE
Updated Go module-3 based on review

### PR DESCRIPTION
**Pull request by Doug-AWS**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: module-3/app/client/mysfitsTableClient.go [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+    ms := Mysfits{}
+
+    err := dynamodbattribute.UnmarshalListOfMaps(items, &ms)
+    if err != nil {
+        return ""
+    }
+
+    return ms.toJson()
+
+    /*
```
Comment (dave-malone) [inline module-3/app/client/mysfitsTableClient.go]: Remove commented code here, lines 250-281

File: module-3/app/service/mythicalMysfitsService.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+    "os"
+
+    L "../client"
```
Comment (dave-malone) [inline module-3/app/service/mythicalMysfitsService.go]: Relative path imports not recommended. Use full package imports instead. Let me know if you need help with this in your branch as there's a bit of a trick to it. Consider using a more descriptive alias for the client than `L`

File: module-3/app/service/mythicalMysfitsService.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+			items = L.QueryMysfits(filter, value)
+		}
+	} else {
+		items = L.GetAllMysfits()
+	}
+
+	// Add items to web page
+	body := []byte(items)
+
+	// Let the web server know it's JSON
```
Comment (dave-malone) [inline module-3/app/service/mythicalMysfitsService.go]: Clean up commented code lines 42-46

File: module-3/app/service/mythicalMysfitsService.go (hunk 2 above)
Comment (Doug-AWS) [reply module-3/app/service/mythicalMysfitsService.go]: I'm not sure how to change this to use a full package import.

**Pull request merged**